content/guides/getting-started/writing-your-first-component-test.md

```diff
@@ -14,6 +14,8 @@ title: Writing Your First Component Test
 
 </Alert>
 
+<CtBetaAlert></CtBetaAlert>
+
```

Comment:
To be consistent this banner should go at the very top of the content, so before the "what you'll learn block"